bosh/init.sh

```
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Can we add a set -e to fail if something is missing?